src/main/resources/assets/ironchest/lang/fi_FI.lang

```diff
@@ -0,0 +1,177 @@
+################
+#    Chests    #
+################
+
+tile.ironchest.chest.iron.name=Rautainen arkku
+tile.ironchest.chest.gold.name=Kultainen arkku
+tile.ironchest.chest.diamond.name=Timanttinen arkku
+tile.ironchest.chest.copper.name=Kupari arkku
+tile.ironchest.chest.silver.name=Hopea arkku
+tile.ironchest.chest.crystal.name=Kristallinen arkku
+tile.ironchest.chest.obsidian.name=Obsidiaani arkku
+tile.ironchest.chest.dirtchest9000.name=Multa arkku 9000!
```

Comment:
Maybe using one style for all chests is better?

Also compound words: Kupari arkku, Hopea arkku, Obsidiaani arkku and Multa arkku 9000!

Reply:
I do translate for better listening in finnish and I asking from finnish friend when I do this translates

Reply:
I admit in this case maintaining one style might be hard.

E.g. Kristalliarkku is better than Kristallinen arkku and Obsidiaaniarkku is better than Obsidiaani arkku or Obsidiaaninen arkku.

Reply:
Ok. You from finland and you now better